railties/lib/rails/commands/destroy/destroy_command.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "rails/generators"
+require_relative "destroy_assistance"
```

Comment:
@AdamMusa I think a private method in this class will suffice. Also, we generally don't use `require_relative` afaik